code/game/area/areas.dm

```diff
@@ -62,6 +62,9 @@
 	var/station_area = FALSE
 	var/centcomm_area = FALSE
 
+	var/narrate //A text-based description of what this area is for.
```

Comment:
Documentation should be like written like this:
```
/// documentation here above the thing
var/thing_or_function_or_whatever
```

Reply:
This should be a more descriptive var name than `narrate`, which is a verb. Name it something like `blurb`